src/js/extensions/auto-link.js

```diff
@@ -30,6 +30,8 @@ LINK_REGEXP_TEXT =
 
     AutoLink = Extension.extend({
         init: function () {
+            Extension.prototype.init.apply(this, arguments);
```

Comment:
Is this something like *calling the parent* in case that if in the future the default `init` function in the parent may actually do something, it will be automatically done by each children?
Because at the moment, `Extension.prototype.init` doesn't do anything: `init: function () {},`